test/e2e/testsuites/testsuites.go

```diff
@@ -1208,6 +1199,31 @@ func (t *TestAzVolumeAttachment) WaitForReplicas(numReplica int, timeout time.Du
 	return wait.PollImmediate(time.Duration(15)*time.Second, timeout, conditionFunc)
 }
 
+func CreateNodeFailure(client v1alpha1ClientSet.DiskV1alpha1Interface, namespace, nodeName string) error {
```

Comment:
This method is not significantly different from `DeleteTestAzDriverNode`, why not update and reuse that method to avoid duplication.

Reply:
Good point. I forgot about the existing functions